src/eradius_proxy.erl

```diff
@@ -39,25 +50,25 @@
                           {timeout, ?DEFAULT_TIMEOUT},
                           {retries, ?DEFAULT_RETRIES}]).
 
--type route() :: eradius_client:nas_address().
--type routes() :: [{Name :: string(), route()}].
+-type route() :: {default_route, eradius_client:nas_address()}.
```

Comment:
Looks like need update `eradius_client:nas_address()`
```erlang
-type nas_address() :: {string() | binary() | inet:ip_address(),  eradius_server:port_number(),  eradius_lib:secret()}
                     | {string() | binary() | inet:ip_address(),  eradius_server:port_number(),  eradius_lib:secret(), PoolName :: atom()}.
```

Reply:
pool name is not related to nas_address()